### PR TITLE
feat(ffe-form-react): Added new prop labelClass to RadioBlock.

### PR DESCRIPTION
**Pull request by cecilieboe**

### CODE REVIEW

**Review by ivarni — APPROVED** [review]

File: packages/ffe-form-react/src/RadioBlock.js [hunk 1]
```diff
@@ -40,7 +41,10 @@ class RadioBlock extends Component {
                 />
                 <div className="ffe-radio-block__content">
                     <label
-                        className="ffe-radio-block__header"
+                        className={classNames(
+                            'ffe-radio-block__header',
+                            className,
```
Comment (ivarni) [inline packages/ffe-form-react/src/RadioBlock.js]: s/className/labelClass/

**Review by ivarni — CHANGES_REQUESTED** [review]

File: packages/ffe-form-react/src/RadioBlock.js (hunk 1 above)
Comment (cecilieboe) [reply packages/ffe-form-react/src/RadioBlock.js]: Fikset!

**Review by ivarni — DISMISSED** [review]

**Review by hdaljord — CHANGES_REQUESTED** [review]
Kan du legge inn labelClass?: string i RadioBlockProps i index.d.ts også?

**Comment by cecilieboe** [comment]
> Kan du legge inn labelClass?: string i RadioBlockProps i index.d.ts også?

can do!

**Review by hdaljord — APPROVED** [review]

**Pull request merged**